test/e2e/Core_view.spec.js

```diff
@@ -118,6 +118,21 @@ describe('Core_view', () => {
 
   });
 
+  it('should scroll viewport to the correct column', async() => {
```

Comment:
Maybe we can use this test spec to prevent potential bugs with vertical scrolling also?

```suggestion
  it('should scroll viewport to the last cell in the last row', async() => {
```